src/Playwright/API/Generated/Options/BrowserContextRunAndWaitForConsoleMessageOptions.cs

```diff
@@ -62,6 +64,14 @@ public BrowserContextRunAndWaitForConsoleMessageOptions(BrowserContextRunAndWait
     /// </summary>
     [JsonPropertyName("timeout")]
     public float? Timeout { get; set; }
+
+    /// <summary>
+    /// <para>
+    /// Cancellation token to provide cancellation of waiting for console message
+    /// </para>
+    /// </summary>
+    [JsonPropertyName("cancellationToken")]
+    public CancellationToken CancellationToken { get; set; }
```

Comment:
Did you add these manually? All these files in that folder are generated automatically. We should add csharp specific properties upstream [like here](https://github.com/search?q=repo%3Amicrosoft%2Fplaywright+RunAndWaitFor+language%3AMarkdown&type=code) so that the generator will pick it up.

Reply:
Yes, manual addition. I will move it to the source for the generator.

Reply:
See [here](https://github.com/microsoft/playwright-dotnet/blob/main/CONTRIBUTING.md#generating-the-apis--rolling-the-driver-upstream) for some notes on how it works.

Reply:
https://github.com/microsoft/playwright/pull/23953